src/styles/blocks/header.scss

```
@@ -38,5 +38,9 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
+    &:hover {
```

add a transition-property
```suggestion
    transition: 300ms;
    
    &:hover {
```